### PR TITLE
vim-checkbox-movements: implemented

### PR DESCRIPTION
**Pull request by tennysontbardwell**
added a new function to interpret ascii keys in the checkbox file, if
the key is j or k it is remapped to the up/down function

### CODE REVIEW

**Comment by tennysontbardwell** [comment]
Really wanted this for myself, hope you consider merging it for others!

**Comment by kahkhang** [comment]
Thanks for your contributions @tennysontbardwell ! Do you mind running `python build.py` to generate the consolidated bash scripts before we merge this in?

File: dev/checkbox_input.sh [hunk 1]
```diff
@@ -137,6 +137,19 @@ remove_checkbox_instructions() {
...
+# for vim movements
+on_checkbox_input_ascii() {
+  local key=$1
+  case $key in
+    "j" ) on_checkbox_input_down;;
+    "k" ) on_checkbox_input_up;;
+  esac
+}
+
+_null() {
```
Comment (kahkhang) [inline dev/checkbox_input.sh]: Could you remove this and use the `on_default` function already in `inquirer_common.sh`? Thanks!

File: dev/checkbox_input.sh [hunk 2]
```diff
@@ -177,7 +190,7 @@ _checkbox_input() {
     tput cuu1
   done
 
-  on_keypress on_checkbox_input_up on_checkbox_input_down on_checkbox_input_space on_checkbox_input_enter
+  on_keypress on_checkbox_input_up on_checkbox_input_down on_checkbox_input_space on_checkbox_input_enter _null _null on_checkbox_input_ascii
```
Comment (kahkhang) [inline dev/checkbox_input.sh]: Change `_null` to `on_default`

File: dist/checkbox_input.sh [hunk 3]
```diff
@@ -239,6 +239,19 @@ remove_checkbox_instructions() {
...
+# for vim movements
+on_checkbox_input_ascii() {
+  local key=$1
+  case $key in
+    "j" ) on_checkbox_input_down;;
+    "k" ) on_checkbox_input_up;;
+  esac
+}
+
+_null() {
```
Comment (kahkhang) [inline dist/checkbox_input.sh]: Please remove this too

File: dist/checkbox_input.sh [hunk 4]
```diff
@@ -279,7 +292,7 @@ _checkbox_input() {
     tput cuu1
   done
 
-  on_keypress on_checkbox_input_up on_checkbox_input_down on_checkbox_input_space on_checkbox_input_enter
+  on_keypress on_checkbox_input_up on_checkbox_input_down on_checkbox_input_space on_checkbox_input_enter _null _null on_checkbox_input_ascii
```
Comment (kahkhang) [inline dist/checkbox_input.sh]: And change `_null` to `on_default`

**Review by kahkhang — CHANGES_REQUESTED** [review]
Do you mind addressing these comments as well? Thanks!

**Comment by tennysontbardwell** [comment]
Good catch @kahkhang ! I cleaned it up.

Btw, I did run `python build.py` before the first commit, it generated `dist/`. Please tell me if I didn't do it correctly!

**Comment by kahkhang** [comment]
🎉 Thanks so much @tennysontbardwell ! LGTM

**Pull request merged**